gp-templates/translation-row.php

```diff
@@ -149,17 +149,17 @@
 					<?php if ( $t->translation_status ): ?>
 						<?php if ( $can_approve_translation ) : ?>
 							<?php if ( 'current' !== $t->translation_status ) : ?>
-							<button class="approve" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $t->id ) ); ?>"><strong>+</strong> <?php _e( 'Approve', 'glotpress' ); ?></button>
+							<button class="approve" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $t->id ) ); ?>" title="<?php echo esc_attr( __( 'Approve this translation.  Any existing translation will be kept as part of the translation history.', 'glotpress' ) ); ?>"><strong>+</strong> <?php _e( 'Approve', 'glotpress' ); ?></button>
```

Comment:
Can you replace `echo esc_attr( __( )` with `esc_attr_e()`? And the double whitespace after the fullstop should be removed.

Reply:
Done.